osmosis/assetlist.json

```diff
@@ -14261,6 +14261,26 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/bOSMO.png"
         }
       ]
+    },
+    {
+      "description": "Memecoin for The International Brane Wave",
+      "denom_units": [
+        {
+          "denom": "factory/osmo13gu58hzw3e9aqpj25h67m7snwcjuccd7v4p55w/brnz",
+          "exponent": 0
+        },
+        {
+          "denom": "brnz",
+          "exponent": 0
+        }
+      ],
+      "base": "factory/osmo13gu58hzw3e9aqpj25h67m7snwcjuccd7v4p55w/brnz",
+      "name": "BRNZ",
```

Comment:
It is recommended to have the name be as the token should be pronounced in conversation. Should users spell out BRNZ, or should they say 'Brainz' (which is just my guess of what this is supposed to be called)?

Reply:
Pronounced brainz, spelled anyway they want

Reply:
Goal is for the ticker to look like branez or burnz, just changed the name tho